src/snowflake/connector/auth/_auth.py

```diff
@@ -411,20 +415,38 @@ def post_request_wrapper(self, url, headers, body) -> None:
             )
         else:
             logger.debug(
-                "token = %s", "******" if ret["data"]["token"] is not None else "NULL"
+                "token = %s",
+                "******"
+                if ret["data"] and ret["data"]["token"] is not None
```

Comment:
Should we use `ret["data"].get("token")`, in case `"token"` does not exist?

Reply:
updated